va/proto/va.proto

```diff
@@ -43,3 +44,11 @@ message ValidationResult {
   string perspective = 3;
   string rir = 4;
 }
+
+message ValidationRequest {
+  core.Identifier identifier = 1;
+  core.Challenge challenge = 2;
+  int64 regID = 3;
+  string authzID = 4;
+  string keyAuthorization = 5;
```

Comment:
> In PerformValidationRequest we call this `expectedKeyAuthorization`, which I like since it's clear what it's doing. Mind following the same name here?

Yes, I can do this. I shortened this up so we can keep the method declaration short enough for one line.

> Also, any particular reason to move regID and authzID from fields of AuthzMeta to being directly on the ValidationRequest?

This type is never re-used by being passed to something further down the chain, it seems like we're just using it to nest two values. This isn't a pattern that we follow anywhere else either. It seems like a good opportunity to flatten it.